src/core/selectors.ts

```diff
@@ -561,6 +561,14 @@ namespace Selectors {
             ? { navigationName: nav.field, type: 'Range', high, low }
             : { navigationName: nav.field, type: 'Value', value }), [])), []);
 
+  /**
+   * Helper function to display record count limited by the max records we can
+   * return from the database.
+   */
+  export const getLimitedCountDisplay = (count: number): string => {
+    const limitedCount = Search.extractRecordCount(count);
+    return count <= limitedCount ? `${count}` : `${limitedCount}+`;
```

Comment:
As much as I feel that a template literal with just a variable in it feels wrong, the alternative is `count + ''` or `count.toString()`, so I'm okay with it.